io.catenax.transmission.transmission_pass/1.0.0/TransmissionPass.ttl

```diff
@@ -0,0 +1,452 @@
+    bamm-c:constraint :MaxTitleCharacters.
+:MaxTitleCharacters a bamm-c:LengthConstraint;
+    bamm:preferredName "Max title characters"@en;
+    bamm:description "Maximum title characters set to 60."@en;
+    bamm-c:maxValue "60"^^xsd:nonNegativeInteger.
+:productStatusValue a bamm:Property;
+    bamm:preferredName "product status value"@en;
+    bamm:description "Status of the transmission as first life or second life."@en;
+    bamm:characteristic :ProductStatusEnumeration;
+    bamm:exampleValue "second life".
+:remanufacturingDate a bamm:Property;
+    bamm:preferredName "remanufacturing date"@en;
+    bamm:description "Date of the (re-)manufacturing"@en;
+    bamm:characteristic bamm-c:Timestamp.
+:ProductStatusEnumeration a bamm-c:Enumeration;
+    bamm:preferredName "Product Status Enumeration"@en;
+    bamm:description "Enumeration to describe the status of the transmission as first life or second life."@en;
+    bamm:dataType xsd:string;
+    bamm-c:values ("first life" "second life").
+:driveType a bamm:Property;
+    bamm:preferredName "drive type"@en;
+    bamm:description "The type of transmission drive (combustion engine, hybrid drive or purely electric drive)."@en;
+    bamm:characteristic :DriveTypeEnumeration;
+    bamm:exampleValue "hybrid drive".
+:torque a bamm:Property;
+    bamm:preferredName "torque performance"@en;
+    bamm:description "The maximum input torque in newton meters (the operating characteristics of the transmission)."@en;
+    bamm:characteristic :TorqueNewtonMetre;
+    bamm:exampleValue "500.0"^^xsd:decimal;
+    bamm:see <https://eclass.eu/eclass-standard/content-suche/show?tx_eclasssearch_ecsearch[cc2prdat]=0173-1#02-ABE835#001&tx_eclasssearch_ecsearch[discharge]=0&tx_eclasssearch_ecsearch[id]=-1&tx_eclasssearch_ecsearch[language]=1&tx_eclasssearch_ecsearch[option]=pr&tx_eclasssearch_ecsearch[searchtxt]=0173-1#02-ABE835#001&tx_eclasssearch_ecsearch[version]=12.0&cHash=4a7d386fbdca6eb1be51134bfeafeac7>.
+:power a bamm:Property;
+    bamm:preferredName "power"@en;
+    bamm:description "The maximum power in kilowatt (the operating characteristics of the transmission)."@en;
+    bamm:characteristic :PerformancePower;
+    bamm:exampleValue "300.0"^^xsd:decimal.
+:torqueConverter a bamm:Property;
+    bamm:preferredName "torqueConverter"@en;
+    bamm:description "A list of the used transmission damper technology (depending on the transmission type)."@en;
+    bamm:characteristic :StringList;
+    bamm:exampleValue "{NW 200 TTD; NW 200 ZDW}".
+:speedResistance a bamm:Property;
+    bamm:preferredName "speed resistance"@en;
+    bamm:description "The permissible maximum input speed in the respective gear (depending on the transmission type). "@en;
+    bamm:characteristic :StringList;
+    bamm:exampleValue "{1 : max 7800min-1; 2 : max 7800min-1; 3 : max 7800min-1; 4 : max 7800min-1; 5 : max 7800min-1; 6 : max 7800min-1; 7 : max 6800min-1; 8 : max 5800min-1; R : max 2000min-1}".
```

Comment:
same here

Reply:
structured value

Reply:
oder in zwei prorperties splitten (vorne mit regex)